online-inference/tensorizer-isvc/tensorizer_hf_isvc/load_model.py

```diff
@@ -0,0 +1,60 @@
+import time
+
+import torch
+from tensorizer import TensorDeserializer
+from tensorizer.utils import convert_bytes, get_mem_usage, no_init_or_tensor
+from transformers import AutoConfig, AutoModelForCausalLM, GPTJForCausalLM
+
+# GLOBALS
+
+DEVICE = "cuda"
+
+"""
+Loads the model using Tensorizer or Huggingface.
+
+params:
+- model_load_type (str): Type of model load [Options: "tensorizer", "hf"]
+"""
+
+
+def load_model_based_on_type(model_load_type: str = "tensorizer"):
+    assert model_load_type == "hf" or model_load_type == "tensorizer"
+
+    if model_load_type == "hf":
+        start = time.time()
+        model = GPTJForCausalLM.from_pretrained(
+            "/mnt/pvc", torch_dtype=torch.float16
+        ).to(DEVICE)
+        print(f"Start time for model load from HF: {time.time() - start} seconds")
+
+        return model
+
+    else:
+        model_ref = "EleutherAI/gpt-j-6B"
+        config = AutoConfig.from_pretrained(model_ref)
```

Comment:
nit: you don't need the `else` here bc you're returning in the `if`, so can save on another level of indentation.